From 7c86b76caf3d1d28fb356bd8f2ae2a3be6c0adc0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Tue, 18 Mar 2025 21:01:10 +0100
Subject: [PATCH 04/11] Revert "ci: Disable sysext temporarily"

gitlab.freedesktop.org is back, so we can re-enable the
jobs that depend on it.

This reverts commit 46b4519361f02ac86e50647e2cc1a7d13f99e8d9.

Part-of: <https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/4343>
---
 .gitlab-ci.yml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/.gitlab-ci.yml b/.gitlab-ci.yml
index 20abfa287..5e33496d0 100644
--- a/.gitlab-ci.yml
+++ b/.gitlab-ci.yml
@@ -21,7 +21,6 @@ build-sysext:
   # https://gitlab.gnome.org/Infrastructure/Infrastructure/-/issues/1665
   tags:
     - x86_64_v3
-  when: manual
 
 stages:
  - review
-- 
2.47.1

